packages/cadl-ranch-specs/http/clients/interfaces/mockapi.ts

```diff
@@ -0,0 +1,54 @@
+import { passOnSuccess, mockapi, json } from "@azure-tools/cadl-ranch-api";
+import { ScenarioMockApi } from "@azure-tools/cadl-ranch-api";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+
+Scenarios.MultiInterfaceClient_dogs_getDogs = passOnSuccess(
```

Comment:
What scenario names do you expect to have? I think `clients\interfaces` is to validate different interfaces can have different routes.

Reply:
then the scenarios should be describing that. If someone looks at the coverage dashboard and see that a generator support or doesn't `getDogs` that means absolutely nothing.

Scenario don't need to be on operation they can be on Namespace too so that you can have a scenario where you call multiple operations. But I guess I can also allow putting in on interface. So we could have a scenario `OperationGroup_group1` and `OperationGroup_group2`

Also note that I am not sure if the goal is to test operation group that this spec is up to date with the latest decision on what operation group should be in cadl.